src/Compute/Compute/VirtualMachine/Operation/NewAzureVMCommand.cs

```diff
@@ -59,6 +59,7 @@
 
 namespace Microsoft.Azure.Commands.Compute
 {
+    [GenericBreakingChangeWithVersion("In the next breaking change period (Nov 2025), the default VM size will change from 'Standard_D2s_v3' to 'Standard_D2s_v5'.", "15.0.0", "11.0.0")]
```

Comment:
https://github.com/Azure/azure-powershell-common/blob/main/src/Common/CustomAttributes/GenericBreakingChangeWithVersionAttribute.cs#L57
Use this overload of GenericBreakingChangeWithVersion for a format of warning message that's consistent with others
```suggestion
    [GenericBreakingChangeWithVersion("The default VM size will change from 'Standard_D2s_v3' to 'Standard_D2s_v5'.", "15.0.0", "11.0.0", "Nov 2025")]
```